jwtauth.go

```diff
@@ -88,6 +89,15 @@ func (ja *JwtAuth) Handle(paramAliases ...string) func(chi.Handler) chi.Handler
 				return
 			}
 
+			// Check expiry via "exp" claim
+			if exp, ok := token.Claims["exp"].(int64); ok {
```

Comment:
Claims are unmarshalled from `map[string]interface{}` - this will never be `int64`. It will be `float64` with default parrser or `json.Number` if `jwt.Parser{UseJSONNumber: true}` is used.